components/automate-ui/src/app/modules/infra-proxy/environment-details/environment-details.component.html

```diff
@@ -31,8 +31,10 @@
         <app-edit-environment-attribute-modal
           [jsonText]="jsonText"
           [openEvent]="openEnvironmentModal"
-          [serverId]="serverId" [orgId]="orgId"
-          [name]="name" [environment]="environment"
+          [serverId]="serverId"
+          [orgId]="orgId"
+          [name]="name"
+          [environment]="environment"
           [label]="label"
           [cookbookConstraints]="cookbookConstraints"
           [cookbookVersions]="cookbookVersions">
```

Comment:
Author note: clarity -- do not hide some attributes by putting two on a line just in a couple cases. If short enough, OK to put all on one line. Otherwise, each on a line by itself.